components/builder-sessionsrv/src/migrations/accounts.rs

```diff
@@ -113,5 +113,26 @@ pub fn migrate(migrator: &mut Migrator) -> SrvResult<()> {
                  ) RETURNS void AS $$
                         DELETE FROM account_origins WHERE account_name=aod_account_name AND origin_id=aod_origin_id;
                  $$ LANGUAGE SQL VOLATILE"#)?;
+
+    // This query is a report that's meant to be run manually. There's no code path (as of
+    // 2017-10-09) that calls it
+    migrator.migrate("accountsrv",
+                     r#"CREATE OR REPLACE FUNCTION account_creation_report (
+                          op_date timestamptz
+                        ) RETURNS TABLE(email text, created_at timestamptz) AS $$
```

Comment:
We'll need their login, too. Email is now an optional field after we switched to a GitHub App. The user can hide their email from us now

Reply:
Ok, will add.